flashCards.rb

```diff
@@ -0,0 +1,123 @@
+require "pry"
+class Flashcard 
+	attr_accessor :front, :back
+	def initialize
+
+	@front = "front"
+    @back = "back"
+    @new_front = []
+    @new_back = []
+
+    @@default_cards = [@front, @back]
+    @@new_cards = [@new_front, @new_back]
```

Comment:
After you take it out of initialize method, you can have `@@new_cards = []`, and then in `initialize` add `@@new_cards.push(self)`.